Description: Disable the JDiff report (not in Debian and not necessary)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -1142,7 +1142,7 @@
 	apply plugin: "groovy"
 
 	// apply plugin: "detect-split-packages"
-	apply from: "${gradleScriptDir}/jdiff.gradle"
+	//apply from: "${gradleScriptDir}/jdiff.gradle"
 
 	asciidoctor {
 		sourceDir project.file('src/asciidoc')
